### PR TITLE
Added known issues on master onprem-ent

### PR DESCRIPTION
**Pull request by furqanmlk**

### CODE REVIEW

File: data/known_issue/onprem-ent.json [hunk 1]
```diff
@@ -1,296 +1,86 @@
...
     },
     {
         "spec_file": "tests/integration/collapsed_reply_threads/last_viewed_spec.js",
         "cases": [
             {
-                "title": "Collapsed Reply Threads > MM-T4843_2 should go to threads view when threads view is the penultimate view and leave the current channel",
+                "title": "Collapsed Reply Threads > MM-T4843_1 should go to threads view when switching a team if that was the last view on that team",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:18]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 2]
```diff
@@ -1,296 +1,86 @@
...
     {
-        "spec_file": "tests/integration/emoji/sorted_emojis_spec.js",
+        "spec_file": "tests/integration/integrations/builtin_commands/invite_command_spec.js",
         "cases": [
             {
-                "title": "Emoji sorting > MM-T157 Filtered emojis are sorted by recency, then begins with, then contains (alphabetically within each)",
+                "title": "Integrations > MM-T660_4 /invite tests when used in DMs and GMs",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:28]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 3]
```diff
@@ -1,296 +1,86 @@
...
-            {
-                "title": "MM-T2575 Extend Session - Email Login > with SessionLengthWebInHours 24 and threshold met, should extend session on visit to a channel",
-                "is_known": true,
-                "type": "known"
-            },
-            {
-                "title": "MM-T2575 Extend Session - Email Login > with SessionLengthWebInHours 74 and threshold met, should extend session on visit to a channel",
-                "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:38]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 4]
```diff
@@ -1,296 +1,86 @@
...
-    },
-    {
-        "spec_file": "tests/integration/multi_team_and_dm/close_current_dm_redirects_spec.js",
-        "cases": [
-            {
-                "title": "Direct messages: redirections > MM-T453_2 Closing a different direct message should not affect active direct message",
+                "title": "Keyboard Shortcuts > MM-T1230 Alt/Option + Down",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:43]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 5]
```diff
@@ -1,296 +1,86 @@
...
     {
-        "spec_file": "tests/integration/multi_team_and_dm/gm_header_spec.js",
+        "spec_file": "tests/integration/messaging/message_reply_bot_post_spec.js",
         "cases": [
             {
-                "title": "Multi-user group header > MM-T473_2 Edit GM channel header (2/2)",
+                "title": "Messaging > MM-T93 Replying to an older bot post that has no post content and no attachment pretext",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:53]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 6]
```diff
@@ -1,296 +1,86 @@
...
     {
-        "spec_file": "tests/integration/multi_team_and_dm/join_open_team_from_dm_spec.js",
+        "spec_file": "tests/integration/multi_team_and_dm/multi_team_join_spec.js",
         "cases": [
             {
-                "title": "Join an open team from a direct message link > MM-T452 User with no teams should be able to join an open team from a link in direct messages",
+                "title": "Multi-Team + DMs > MM-T1805 No infinite loading spinner on Select Team page",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:63]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 7]
```diff
@@ -1,296 +1,86 @@
...
     {
-        "spec_file": "tests/integration/onboarding/login_page_link_account_creation_spec.js",
+        "spec_file": "tests/integration/plugins/marketplace/render_in_main_menu_spec.js",
         "cases": [
             {
-                "title": "Onboarding > MM-T400 Create account from login page link using email-password",
+                "title": "Plugin Marketplace > MM-T1960 Marketplace is available when \"Enable Plugins\" is true",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:73]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json [hunk 8]
```diff
@@ -1,296 +1,86 @@
...
-                "title": "M17448 Does not post draft message > on successful upload via \"Draw\" plugin",
-                "is_known": true,
-                "type": "known"
-            },
-            {
-                "title": "M17448 Does not post draft message > on upload cancel via \"Draw\" plugin",
+                "title": "SupportSettings > MM-T1034 - Customization: Blank TOS link field (About modal)",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (mattermost-quality) [inline data/known_issue/onprem-ent.json:83]:
```suggestion
                "type": "flaky"
```

**Review by mattermost-quality — CHANGES_REQUESTED** [review]
.

File: data/known_issue/onprem-ent.json [hunk 9]
```diff
@@ -1,296 +1,86 @@
...
     {
-        "spec_file": "tests/integration/accessibility/accessibility_post_spec.js",
+        "spec_file": "tests/integration/channel/new_channel_with_board_spec.js",
         "cases": [
             {
-                "title": "Verify Accessibility Support in Post > MM-T1486_3 Verify Tab support on Post on Center Channel",
+                "title": "New Channel modal with Boards enabled > MM-T5141 New Channel is created with an associated Board",
                 "is_known": true,
-                "type": "known"
+                "type": "knwon"
```
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]: For now, let's have the type either flaky and bug. Other could be test infra/server limitation but not sure what's the short term for it.
If this is flaky, then:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 1 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 2 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 3 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 4 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 5 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 6 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 7 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

File: data/known_issue/onprem-ent.json (hunk 8 above)
Comment (saturninoabril) [inline data/known_issue/onprem-ent.json]:
```suggestion
                "type": "flaky"
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @furqanmlk, LGTM except for changing the type so that in UI it will present better unlike in the screenshot.
Screenshot 2023-01-17 at 7 14 19 AM

**Review by saturninoabril — APPROVED** [review]

**Pull request merged**